querydsl-apt/src/test/java/com/querydsl/apt/DateExtensionsTest.java

```diff
@@ -34,7 +34,7 @@
     private static final String packagePath = "src/test/java/com/querydsl/apt/";
 
     @Test
-    public void Handles_Date_Extensions_Correctly() throws IOException, InterruptedException {
+    public void handles_Date_Extensions_Correctly() throws IOException, InterruptedException {
```

Comment:
I understand the 'getting rid of capitalized test method names' part.
I do think that the PR should maybe be 'unify naming of method names'.
If we go from one non-standard way into another, that might distract from the development.